tests/e2e/constants/TimeoutConstants.ts

```diff
@@ -226,6 +226,11 @@ export const TimeoutConstants: any = {
     /**
      * Common timeout for plugins.
      */
-    TS_COMMON_PLUGIN_TEST_TIMEOUT: Number(process.env.TS_COMMON_PLUGIN_TEST_TIMEOUT) || 30_000
+    TS_COMMON_PLUGIN_TEST_TIMEOUT: Number(process.env.TS_COMMON_PLUGIN_TEST_TIMEOUT) || 30_000,
 
+    /**
+     * Timeout waiting for disappearance of Two Factor Authentication info box, "360 000" by default
+     */
+    TS_SELENIUM_GIT_TWO_FACTOR_TIMEOUT:
+        Number(process.env.TS_SELENIUM_GIT_TWO_FACTOR_TIMEOUT) || 360_000,
```

Comment:
Is this timeout should be so long?

Reply:
You added this timeout into plugin section. Can you move it up, or add some comment?

Reply:
@Daybringer : timeouts in tests should be as minimal as possible to fail test quickly if there is nothing to wait, and so have test result as soon as possible, and decrease cost of test execution.

Which maximum time of waiting for disappearance of Two Factor Authentication info box did you see when run the test?

Reply:
@dmytro-ndp Mentioned in https://github.com/eclipse/che/pull/22202/commits/c8885d3dd2977d26e6ae5839ec7eb29e5dc8fdda